### PR TITLE
Add missing uninstall.sh scripts

### PR DESCRIPTION
**Pull request by SorokinaVeronika**
## BACKGROUND
Each stack in the marketplace-kubernetes repo should have an uninstall.sh file

-----------------------------------------------------------------------

## Changes
* Added missing uninstall.sh scripts to all stacks

-----------------------------------------------------------------------

## Checklist
- [ ] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/jaeger/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+################################################################################
+# chart
+################################################################################
+STACK="jaeger"
+NAMESPACE="jaeger"
+
+
+helm uninstall "$STACK" \
+  --namespace "$NAMESPACE"
```
Comment (mbeyt) [inline stacks/jaeger/uninstall.sh:13]: Deploy error:
```Error: INSTALLATION FAILED: repo jaegertracing not found```

File: stacks/mattermost-operator/uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+kubectl delete --ignore-not-found=true namespace mysql-operator
+kubectl delete --ignore-not-found=true namespace minio-operator
+kubectl delete --ignore-not-found=true namespace mattermost-operator
```
Comment (mbeyt) [inline stacks/mattermost-operator/uninstall.sh:7]: New line

File: stacks/rstudio/uninstall.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+kubectl delete --ignore-not-found=true namespace rstudio
```
Comment (mbeyt) [inline stacks/rstudio/uninstall.sh:5]: new line

File: stacks/shiny/uninstall.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+kubectl delete --ignore-not-found=true namespace shiny
```
Comment (mbeyt) [inline stacks/shiny/uninstall.sh:5]: new line

File: stacks/victoria-metrics-cluster/uninstall.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  ROOT_DIR=$(git rev-parse --show-toplevel)
+  VMCLUSTER="$ROOT_DIR/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml"
+  VMAGENT="$ROOT_DIR/stacks/victoria-metrics-cluster/yaml/vmagent.yaml"
+  kubectl delete -f "$VMCLUSTER"
+  kubectl delete -f "$VMAGENT"
+else
+  # with github hosted master version of values.yml
+  kubectl delete -n default -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml
+  kubectl delete -n default -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmagent.yaml
+fi
```
Comment (mbeyt) [inline stacks/victoria-metrics-cluster/uninstall.sh:23]: new line

File: stacks/cloudcasa/uninstall.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+################################################################################
+# chart
+################################################################################
+STACK="cloudcasa"
+NAMESPACE="default"
+
+
+helm uninstall "$STACK" \
+  --namespace "$NAMESPACE"
```
[the remark this answers is not in the document]
Comment (scott) [reply stacks/cloudcasa/uninstall.sh:13]: cloudcasa installed from the marketplace- looking into why the deploy script didn't work for you 🤔

[the remark this answers is not in the document]
Comment (scott) [reply stacks/cloudcasa/uninstall.sh:13]: i got the same error when running the deploy script... wondering if its something to do with the versions of helm.

File: stacks/victoria-metrics-cluster/uninstall.sh (hunk 5 above)
Comment (mbeyt) [inline stacks/victoria-metrics-cluster/uninstall.sh:23]: I don't get errors from this, but it does hang indefinitely and the pods don't get deleted.

**Review by mbeyt — APPROVED** [review]

**Pull request merged**